.github/workflows/ci.yaml

```diff
@@ -146,7 +149,7 @@ jobs:
       - name: "Run tests"
         shell: bash
         # We can't reject unreferenced snapshots on windows because flake8_executable can't run on windows
-        run: cargo insta test --all --all-features
+        run: cargo insta test --all --exclude ruff_dev --all-features
```

Comment:
Can we add another run command that runs the test of `ruff_dev` but with the default features only? We otherwise lose the schema test.